index.Rmd

```diff
@@ -9,7 +9,7 @@ biblio-style: apalike
 link-citations: yes
 github-repo: ropenscilabs/dev_guide
 url: 'https\://ropensci.github.io/dev_guide/'
-description: "Extended version of the rOpenSci packaging guide."
+description: "Extended version of the rOpenSci packaging guide. This book is a guide for authors, maintainers, reviewers and editors of rOpenSci. The first section of the book contains our guidelines for creating and testing R packages. The second section is dedicated to rOpenSci's software peer review process: what it is, our policies, and specific guides for authors, editors and reviewers throughout the process. The third and last section features our best practice for nurturing your package once it has been onboarded: how to collaborate with other developers, how to document releases, how to promote your package and how to leverage GitHub as a development platform. The third section also features a chapter for anyone wishing to start contributing to rOpenSci packages."
 cover-image: images/cover.png
```

Comment:
why does this change appear here?

Reply:
It's not intended. I'll see what's going on. Sorry.

Reply:
np

Reply:
@maelle, I don't know why that text appears where it does. My commit https://github.com/ropensci/dev_guide/pull/169/commits/20eb0b41a420c424c2fca6ec8d0f72abd2d35aa8 doesn't seem to have done it.

What am I missing here?

Reply:
not sure but maybe your dev branch was not up to date? The last contributor name on up-to-date dev is Hugo Gruson https://github.com/ropensci/dev_guide/blob/dev/index.Rmd

Reply:
> maybe your dev branch was not up to date
You are right! I reseted my branch with upstream/dev, then cherry-picked my commit and forced push. It should now be good.

Reply:
Thank you!